shell/platform/darwin/ios/framework/Headers/FlutterAppDelegate.h

```diff
@@ -29,6 +29,15 @@ FLUTTER_EXPORT
 
 @property(strong, nonatomic) UIWindow* window;
 
+/**
+ * Handle StatusBar touches.
+ *
+ * If you want your Flutter views to respond to StatusBar touches, like scroll-to-top behavior, you
+ * need to call this function from your AppDelegate's `touchesBegan:withEvent:`.  FlutterAppDelegate
```

Comment:
method, not function.

Reply:
Technically a "class method", removed it to avoid any confusion since I think people hardly call them class methods.  Most people come from a java or c++ education so they think of them as static functions.